client/app/pages/dashboards/hooks/useDashboard.js

```diff
@@ -144,6 +144,29 @@ function useDashboard(dashboardData) {
     [loadWidget]
   );
 
+  const persistParameterValues = useCallback(
```

Comment:
Now, I think we can move the implementation of persistParameterValues to saveDashboardParameters , as both function is doing one thing and each are short enough.